doc/syntax.rdoc

```diff
@@ -11,6 +11,9 @@ Assignment[rdoc-ref:syntax/assignment.rdoc] ::
 {Control Expressions}[rdoc-ref:syntax/control_expressions.rdoc] ::
   +if+, +unless+, +while+, +until+, +for+, +break+, +next+, +redo+
 
+{Pattern matching}[rdoc-ref:syntax/pattern_matching.rdoc] ::
+  Experimental structure pattern matching and variable binding syntax
```

Comment:
"structural" (e.g., [here](http://blog.higher-order.com/blog/2009/08/21/structural-pattern-matching-in-java/) and [here](https://blog.carbonfive.com/2018/01/08/an-introduction-to-adts-and-structural-pattern-matching-in-typescript/)) ? And probably the word "feature" is missing here, too.

```suggestion
  Experimental structural pattern matching and variables binding feature syntax
```

Reply:
Agreed about "structural" (fixed), but I don't believe "feature" is suitable here (in generic "TOC of syntax elements").